watertap/examples/flowsheets/nf_dspmde/nf_ui.py

```diff
@@ -21,11 +23,27 @@ def export_to_ui():
         do_export=export_variables,
         do_build=build_flowsheet,
         do_solve=solve_flowsheet,
+        get_diagram=get_diagram,
         requires_idaes_solver=True,
+        category=FlowsheetCategory.wastewater,
+        build_options={
+            "Bypass": {
+                "name": "bypass option",
+                "display_name": "With Bypass",
+                "values_allowed": ["false", "true"],
+                "value": "false",
+            },
+            "ConcentrationPolarization": {
+                "name": "ConcentrationPolarization",
+                "display_name": "Concentration Polarization Type",
+                "values_allowed": ["calculated", "none"],
+                "value": "calculated",
+            },
+        },
```

Comment:
We can revisit this in a subsequent PR. I think we mainly want to get the functionality in at this point and refine later (e.g., add other relevant options, change the category as needed, etc.)